app/utils/markdown/index.ts

```diff
@@ -45,7 +45,7 @@ export const getMarkdownTextStyles = makeStyleSheetFromTheme((theme: Theme) => {
         },
         link: {
             color: theme.linkColor,
-            fontFamily: 'OpenSans',
+            fontFamily: 'Metropolis',
```

Comment:
this is not the proper fix, as a link in normal paragraph will now use Metropolis instead of openSans

Reply:
Sorry, i misunderstood the code , now only the heading link font is Metropolis